packages/wells/src/client/baseWellsClient.ts

```diff
@@ -148,3 +162,39 @@ export default class BaseWellsClient {
     return this.http;
   }
 }
+
+export class ConfigureAPI {
+  protected client?: HttpClientWithIntercept;
+  protected project?: String;
+  protected cluster?: String;
+
+  public set setHttpClient(httpClient: HttpClientWithIntercept) {
+    this.client = httpClient;
+  }
+
+  public set setProject(project: String) {
+    this.project = project;
+  }
+
+  public set setCluster(cluster: String) {
+    this.cluster = cluster;
+  }
+
+  protected getPath(baseUrl: string, cursor?: string): string {
+    if (this.project == undefined) {
+      throw new HttpError(400, 'The client project has not been set.', {});
+    }
+    if (this.cluster == undefined) {
+      throw new HttpError(400, 'No cluster has been set.', {});
+    }
+    let path =
+      this.cluster == Cluster.BLUEFIELD
+        ? `/${this.project}${baseUrl}`
+        : `/${this.project}${baseUrl}?env=${this.cluster}`;
+
+    if (cursor) {
+      path += `&cursor=${cursor}`;
+    }
```

Comment:
Does this work if we don't have an env keyword?
I think this URL would be illegal: `/wells&cursor=abc123`.